assets/src/edit-story/components/devTools/devTools.js

```diff
@@ -55,36 +57,53 @@ const Options = styled.div`
 `;
 
 const Label = styled.label`
-  margin-right: 10px;
+  margin-right: 6px;
 `;
 
+const Button = styled.button`
+  margin-left: 6px;
+`;
+
+const replaceResourcesWithDummy = (state) => {
+  return {
+    ...state,
+    pages: state.pages.map((page) => ({
+      ...page,
+      elements: page.elements.map((element) => {
+        let newElement = { ...element };
+        if ('resource' in element) {
+          newElement.resource = { ...element.resource };
+          newElement.resource.mimeType = 'image/png';
+          newElement.resource.src =
+            'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAIAAAACCAYAAABytg0kAAAAGElEQVQYV2NkYGD4////fwZGEMnIyMgAAEvqB/6wfXLSAAAAAElFTkSuQmCC';
```

Comment:
This is PoC with a dummy image (2x2 checkerboard).
Do we need to replace the resources with random images from the media library? is it that useful?